LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
+
+Copyright (c) 2014 jekyll
```

Comment:
This comes from the website.

Shouldn't we use a more content-oriented license here? CC-by-sa?

Reply:
done